src/components/item/item-sliding.scss

```diff
@@ -80,13 +80,7 @@ ion-item-sliding.active-slide {
     display: flex;
   }
 
-  &.active-options-left ion-item-options[side=left] {
-    width: 100%;
-
-    visibility: visible;
-  }
-
-  &.active-options-right ion-item-options:not([side=left]) {
+  &.active-options ion-item-options {
```

Comment:
the `active-options-left ion-item-options[side=left]` were exists so that we only make it visible when it should be visible; when the client is about to slide the item to the right so the left option should be appear.
with this you will display things when user is about to swap whether left or right! doesn't matter! the options should be appear, in this case we will face a bug when the options are expandable and we have both left and right options and the user swap the item to the left user might see the other options on the left side as well (this can be easily reproducible just put three or four options to the left and one expandable to the right and pull your fingers from right to left) which will consider as a bug, I've spent a full day to figure these out :))